app/controllers/leafs_controller.rb

```diff
@@ -14,10 +14,10 @@ def create
 
     respond_to do |format|
       if @leaf.save
-        flash[:success] = "Leaf created!"
+        flash[:success] = 'Leaf created!'
         format.html { redirect_to new_branch_path }
       else
-        flash[:error] = "Leaf not created for some reason"
+        flash[:error] = 'Leaf not created for some reason'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.